Cocoapods Example/PortalSwift/Portal.swift

```diff
@@ -157,10 +158,21 @@ class PortalWrapper {
     }
   }
 
-  func backup(backupMethod: BackupMethods.RawValue, user: UserResult, backupConfigs: BackupConfigs? = nil, completion: @escaping (Result<Bool>) -> Void) {
+  func backup(backupMethod: BackupMethods.RawValue, user: UserResult, backupConfigs: BackupConfigs? = nil, completion: @escaping (Result<String>) -> Void) {
```

Comment:
Since we're changing the return type, this means adding passkey backup is a breaking change right?